Code/CSharpLINQ/Cat.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics.Metrics;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CSharpLINQ
+{
+    internal class Cat
+    {
+        public string Color { get; set; }
+
+
+        public void Task_09()
```

Comment:
It would be better to move those methods to class Homework as those methods are related to it, not really to class Cat :)

Reply:
I decided to leave those methods in the class Cat